src/editorial-source-components/Tooltip.tsx

```diff
@@ -145,24 +158,23 @@ export const Tooltip = ({ children }: { children: React.ReactNode }) => {
         <SvgInfo />
       </div>
 
-      <div
+      <TooltipBox
         ref={setPopperElement}
         style={styles.popper}
-        css={tooltip}
         data-show={visible || null}
         data-opaque={opaque || null}
         {...attributes.popper}
         onMouseEnter={handleMouseEnter}
         onMouseLeave={handleMouseLeave}
       >
         {children}
-        <div
+        <Arrow
           ref={setArrowElement}
           style={styles.arrow}
-          css={arrow}
+          className={"arrow"}
```

Comment:
Unbearably minor nit: as a string, this can be a literal in the template, we shouldn't need curlies